maquetado/scss/navbar.scss

```diff
@@ -0,0 +1,35 @@
+@import 'variables';
+
+.nav {
+  height: 0;
+  left:0;
+  margin: auto;
+  position: fixed;
+  top: 0;
+  z-index: 9999;
+}
+
+.navbar {
+  background-color: white;
+  border-bottom: 1px lightgray solid;
```

Comment:
I think that this property shouldn't be because with the property `box-shadow` is enough, please delete it